fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/Loan.java

```diff
@@ -779,8 +779,10 @@ private void handleChargePaidTransaction(final LoanCharge charge, final LoanTran
                 .determineProcessor(this.transactionProcessingStrategyCode);
         final List<LoanRepaymentScheduleInstallment> chargePaymentInstallments = new ArrayList<>();
         List<LoanRepaymentScheduleInstallment> installments = getRepaymentScheduleInstallments();
+        int firstNormalInstallmentNumber = LoanRepaymentScheduleProcessingWrapper
+                .fetchFirstNormalInstallmentNumber(repaymentScheduleInstallments);
         for (final LoanRepaymentScheduleInstallment installment : installments) {
-            boolean isDue = installment.isFirstPeriod()
+            boolean isDue = installment.getInstallmentNumber().equals(firstNormalInstallmentNumber)
```

Comment:
I know that it was here before, but I was wondering why `isDue` is the proper variable name here. Imo sameInstallment would be more appropriate isn't it, or am I missing something?